src/main/java/com/taboola/backstage/Backstage.java

```diff
@@ -174,6 +178,16 @@ public BackstageBuilder setWriteTimeoutMillis(Long writeTimeoutMillis) {
             return this;
         }
 
+        public BackstageBuilder setMaxIdleConnections(Integer maxIdleConnections) {
+            this.maxIdleConnections = maxIdleConnections;
+            return this;
+        }
+
+        public BackstageBuilder setKeepAliveDurationSec(Long keepAliveDurationSec) {
```

Comment:
Please make it in Milliseconds to be similar to other things we allow to configure to avoid confusion.

Reply:
OK, changed that.